tests/infer/test_abstract_infer.py

```diff
@@ -31,6 +39,48 @@ def test_posterior_predictive():
     assert_equal(marginal_return_vals.mean, torch.ones(5) * 700, prec=30)
 
 
+def test_posterior_predictive_svi_manual_guide():
+    true_probs = torch.ones(5) * 0.7
+    num_trials = torch.ones(5) * 1000
+    num_success = dist.Binomial(num_trials, true_probs).sample()
+    conditioned_model = poutine.condition(model, data={"obs": num_success})
+    opt = optim.Adam(dict(lr=1e-3))
+    loss = Trace_ELBO()
+    guide = beta_guide
+    svi_run = SVI(conditioned_model, guide, opt, loss, num_steps=10000, num_samples=1000).run(num_trials)
+    posterior_predictive = TracePredictive(model, svi_run, num_samples=10000).run(num_trials)
+    marginal_return_vals = posterior_predictive.marginal().empirical["_RETURN"]
+    assert_equal(marginal_return_vals.mean, torch.ones(5) * 700, prec=30)
```

Comment:
I suggest using `assert_close(rtol=0.05)` (using some suitable value for `rtol`) in these cases, which is easier to read and shows the relative tolerance used at a glance.